recipes/xorg-xft/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set xorg_name = "libXft" %}
+{% set xorg_category = "lib" %}
+{% set name = "xorg-" ~ xorg_name %}
+{% set version = "2.3.2" %}
+{% set sha256 = "f5a3c824761df351ca91827ac221090943ef28b248573486050de89f4bfcdc4c" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 3
+  detect_binary_files_with_prefix: true
+  # required fontconfig is not available on windows:
+  skip: True  # [win]
+
+requirements:
+  build:
+    - pkg-config
+    - toolchain
+    - xorg-libx11 1.6.*
+    - xorg-util-macros
+    - xorg-libpthread-stubs
+    - xorg-libxrender 0.9.*
+    - freetype
```

Comment:
The `freetype` and `fontconfig` lines should specify the version pinnings as listed in [this obscure data file](http://tinyurl.com/je7ypjz), in both the "build" and "runtime" dependency sections.